src/components/Truncate/Truncate.tsx

```diff
@@ -1,9 +1,7 @@
 import * as React from 'react';
 import IReactComponentProps from '../../common/structures/IReactComponentProps';
 import Omit from '../../common/structures/Omit';
-
-/* TruncateMarkup's types and exports are all jacked up. This makes it play nicely in Local. */
-const TruncateMarkup = require('react-truncate-markup').default;
+import * as TruncateMarkup from 'react-truncate-markup/lib';
```

Comment:
So weird how they set this library up. Glad this resolves the issue, however. Great job!